float32 Finger_1
float32 Finger_2
float32 Finger_3
